Xamarin.PropertyEditing.Mac/Controls/CollectionEditorWindow.cs

```diff
@@ -16,8 +16,8 @@ internal class CollectionEditorWindow
 		public CollectionEditorWindow (IHostResourceProvider hostResources, CollectionPropertyViewModel viewModel)
 			: base (new CGRect (0, 0, 500, 400), NSWindowStyle.Titled | NSWindowStyle.Closable | NSWindowStyle.Resizable, NSBackingStore.Buffered, true)
 		{
-			Delegate = new ModalDialogDelegate ();
-			Title = String.Format (Properties.Resources.CollectionEditorTitle, viewModel.Property.Name);
+			Delegate = new ModalWindowCloseDelegate (); //new ModalDialogDelegate ();
```

Comment:
Replacing this usage with a common class that does the same thing is great, but let's remove the old usage and the old class if we're not going to use them.

Reply:
Meant to do that, forgot.